developer_manual/client_apis/files.rst

```diff
@@ -0,0 +1,58 @@
+=====
+Files
+=====
+
+.. sectionauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+
+Since Nextcloud 27, the files app has been rewritten to use new standards
+and frameworks. This means that the old documentation is no longer valid.
+We will update and document the new files app APIs and methods here.
+
+.. note:: Some external libraries offers in-depth technical documentation.
+    Please have a look at the following:
+    * https://github.com/nextcloud/nextcloud-files (`docs <https://nextcloud.github.io/nextcloud-files/>`_)
+    * https://github.com/nextcloud/nextcloud-upload
+
+
+Router
+^^^^^^
+
+To change views and parameters of the current Files app, we expose the router.
+This directly maps to the Vue router. You can check their own `documentation <https://router.vuejs.org/guide/essentials/navigation.html#navigate-to-a-different-location>`_ to
+better understand the methods.
+
+.. code-block:: ts
```

Comment:
does this render as typescript?

Reply:
Seems to yeah

Reply:
PR template asks for a screenshot which would have been helpful to see whether it does:
![grafik](https://github.com/nextcloud/documentation/assets/213943/99ba32f3-a504-4460-8b75-e6da289558e3)